src/lib/message.ts

```diff
@@ -42,6 +45,7 @@ class GleeMessage extends EventEmitter {
    * @param {GleeConnection} [options.connection] The connection through which the message will be sent or has been received.
    * @param {Boolean} [options.broadcast=false] Whether the message should be broadcasted or not.
    * @param {Boolean} [options.cluster=false] Whether the message is from a cluster adapter or not.
+   * @param {Any} [options.query] Message query.
```

Comment:
Please elaborate a bit more on this description. Maybe mention this is just for HTTP query parameters, etc. This doesn't say much right now.